src/components/NetworkSelector/NetworkSelector.js

```diff
@@ -171,13 +177,26 @@ export default function NetworkSelector(props) {
     <div className={cx("Selector", className, { disabled })}>
       {isModalVisible && (
         <div>
-          <Modal className="selector-modal" isVisible={isModalVisible} setIsVisible={toggleModal} label={modalLabel}>
+          <Modal
+            className="selector-modal"
+            isVisible={isModalVisible}
+            setIsVisible={() => toggleModal()}
```

Comment:
does toggleModal use props, the below use does so changing this might effect modal functionality

Reply:
fixed